README.md

```diff
@@ -358,6 +353,18 @@ npm test
 # npm run test-windows
 ```
 
+##### Attribution
+We have an automatic build setup on Travis CI that will run every day to catch changes made by Google as quick as possible. These tests make use of BrowserStack's services to remotely run Selenium tests, huge thanks to [BrowserStack][] for the support of Open Source projects like this one!  
+
+[BrowserStack]: https://www.browserstack.com/
+
+![BrowserStack Logo](https://www.browserstack.com/images/mail/newsletter-bs-logo@2x.png)
```

Comment:
I have a feeling this URL could change on us easily. Let's use the Browserstack logo from https://github.com/gilbarbara/logos

```
![BrowserStack Logo](https://cdn.rawgit.com/gilbarbara/logos/081a89a93b6af2bba5cf4eeace7a5c430785c003/logos/browserstack.svg)
```